ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -131,7 +165,21 @@ router bgp {{ router_bgp.as }}
    neighbor {{ neighbor }} send-community {{ router_bgp.neighbors[neighbor].send_community }}
 {%         endif %}
 {%         if router_bgp.neighbors[neighbor].maximum_routes is arista.avd.defined %}
-   neighbor {{ neighbor }} maximum-routes {{ router_bgp.neighbors[neighbor].maximum_routes }}
+{%             set maximum_routes_cli = "neighbor " ~ neighbor ~ " maximum-routes " ~ router_bgp.neighbors[neighbor].maximum_routes %}
+{%             if router_bgp.neighbors[neighbor].maximum_routes_warning_limit is arista.avd.defined %}
+{%                 set maximum_routes_cli = maximum_routes_cli ~ " warning-limit " ~ router_bgp.neighbors[neighbor].maximum_routes_warning_limit %}
+{%             endif %}
+{%             if router_bgp.neighbors[neighbor].maximum_routes_warning_only is arista.avd.defined(true) %}
+{%                 set maximum_routes_cli = maximum_routes_cli ~ " warning-only" %}
+{%             endif %}
+   {{ maximum_routes_cli }}
+{%         endif %}
+{%         if router_bgp.neighbors[neighbor].allowas_in.enabled is arista.avd.defined(true) %}
+{%             set allowas_in_cli = "neighbor " ~ neighbor ~ " allowas-in" %}
+{%             if router_bgp.neighbors[neighbor].allowas_in.times is arista.avd.defined %}
+{%                 set allowas_in_cli = allowas_in_cli ~ " " ~ router_bgp.neighbors[neighbor].allowas_in.times %}
+{%             endif %}
+   {{ allowas_in_cli }}
```

Comment:
There is no documentation for the allowas-in knob. I think we should include it in the section "BGP tunning" or the peer-group.

Reply:
Hmm don't think the tuning table would be the proper place. 
Under neighbors or the Peer group would make more sense to me.
Question is if the setting is that common, you'd in any case see it via the embedded config snippet..

Reply:
I agree we don't need to document everything in the tables and the configuration is in the snipped. However, in this case, as we are modifying the BGP best path selection algorithm, I believe it is important to add it specifically in the documentation so it is clearly stated.

Reply:
Xavi, adding it to the peer groups like this would make sense I think:

```
| Settings | Value |
| Allowas-in | n |
```

It would also be possible in the neighbors table, as a new column:

```
| Neighbor | Remote AS | VRF | Send-community | Maximum-routes | Allowas-in |
```

I dont see it fit in any of the existing columns..

@ClausHolbechArista @tgodaA whats your view on this?

Reply:
You can add this to neighbors table. Allowas-in with values like Allowed, Allowed 5 times, '-'

I don't have much idea about this but neighbor table looks good to me.

Reply:
@tgodaA Will add it for peer-groups and neighbors